common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1018,6 +1019,39 @@ def has_ended(self):
         """
         return course_metadata_utils.has_course_ended(self.end)
 
+    def enrollment_has_ended(self):
```

Comment:
Since we need this in a single module, I suggest to move this to `edraak_forus.helpers` to avoid future migration issues whenever possible.